src/main/java/core/basesyntax/JavaStreamApi.java

```diff
@@ -85,6 +109,12 @@ public Double averageSumOdd(List<Integer> numbers) {
      * Задача: вивести все имена кошек в которых хозяева это девушки старше 18 лет</p>
      **/
     public List<String> getCatsNames(List<People> peopleList, int femaleAge) {
-        return Collections.emptyList();
+        return peopleList.stream()
+                .filter(x -> x.getSex() == People.Sex.WOMEN && x.getAge() >= femaleAge
+                        && x.getCatList() != null)
```

Comment:
```suggestion
                .filter(x -> x.getSex() == People.Sex.WOMEN && x.getAge() >= femaleAge)
```
we can go without this